grease-cli/src/Grease/Cli.hs

```diff
@@ -345,6 +347,18 @@ symStdinParser =
         <> Opt.help "populate stdin with this many symbolic bytes"
     )
 
+simDumpCoverageParser :: Opt.Parser (Maybe FilePath)
+simDumpCoverageParser =
+  Opt.optional
+    ( Opt.strOption $
+        Opt.long "dump-coverage"
+          <> Opt.metavar "FILE"
+          <> Opt.help
+            ( "Dump the addresses of executed instructions. These addresses are relative to a section"
+                ++ "index which is the first JSON array appended to the file"
+            )
```

Comment:
Would this be a valid [JSON lines](https://jsonlines.org/) file at that point? Perhaps worth saying so, if so.

I like the simplification of a single file, by the way!